c_glib/test/test-record-batch.rb

```diff
@@ -221,5 +221,44 @@ def test_invalid
         end
       end
     end
+
+    sub_test_case("#validate_full") do
+      def setup
+        @id_field = Arrow::Field.new("uint8", Arrow::UInt8DataType.new)
+        @name_field = Arrow::Field.new("string", Arrow::StringDataType.new)
+        @schema = Arrow::Schema.new([@id_field, @name_field])
+
+        @uint8_value = build_uint_array([1])
+        @valid_name_value = build_string_array(["abc"])
+        @n_rows = @uint8_value.length
+
+        # U+3042 HIRAGANA LETTER A, U+3044 HIRAGANA LETTER I
+        data = "\u3042\u3044".b[0..-2]
+        value_offsets = Arrow::Buffer.new([0,data.size].pack("l*"))
+        @invalid_name_value = Arrow::StringArray.new(1,
+                                                    value_offsets,
+                                                    Arrow::Buffer.new(data),
+                                                    nil,
+                                                    -1)
```

Comment:
Should we add a linter/formatter for Ruby...?

```suggestion
                                                     value_offsets,
                                                     Arrow::Buffer.new(data),
                                                     nil,
                                                     -1)
```